app/src/main/java/at/bitfire/icsdroid/ui/TitleColorFragment.kt

```diff
@@ -51,9 +99,18 @@ class TitleColorFragment: Fragment() {
         var originalColor: Int? = null
         val color = MutableLiveData<Int>()
 
-        fun dirty() =
-                originalTitle != title.value ||
-                originalColor != color.value
+        var originalIgnoreAlerts: Boolean? = null
+        val ignoreAlerts = MutableLiveData<Boolean>()
+
+        var originalDefaultAlarmMinutes: Long? = null
+        val defaultAlarmMinutes = MutableLiveData<Long>()
+
+        fun dirty() = arrayOf(
+            originalTitle to title,
+            originalColor to color,
+            originalIgnoreAlerts to ignoreAlerts,
+            originalDefaultAlarmMinutes to defaultAlarmMinutes,
+        ).any { (original, state) -> original != state.value }
```

Comment:
Yeah, both of them work. I changed to this expression because during dev there were like 6 or 7 statements, which made them quite unreadable. With 4 statements `!=` and `||` are more than enough

Reply:
[a68b168](https://github.com/bitfireAT/icsx5/pull/78/commits/a68b1689071791606ecd30a0a94135cb6249a33c) goes back to the old expression